website/projects/awssync.py

```diff
@@ -21,9 +25,36 @@ def button_pressed(self):
 
         :return: True if function executes successfully
         """
-        self.logger.info("Pressed button")
+        print("Pressed button")
+        print(self.get_emails_with_teamids())
```

Comment:
This throws exception `IndexError: list index out of range` @ `project_id = int(str(project_id[0][0]) + str(project_id[0][1]))` upon pressing the button.

Reply:
Think it was because the function didn't check for mailing_lists without projects. Should be solved now.